src/foundry/client/apps/forms/card-config.d.mts

```diff
@@ -36,4 +36,12 @@ declare global {
      */
     protected _onFaceControl(event: JQuery.ClickEvent): Promise<unknown>;
   }
+
+  namespace CardConfig {
+    interface CardConfigData<Options extends DocumentSheetOptions<Card> = DocumentSheetOptions<Card>>
+      extends DocumentSheet.DocumentSheetData<Options, InstanceType<ConfiguredDocumentClassForName<"Card">>> {
+      //TODO: Find if we can better type this
+      types: Record<string, string>;
```

Comment:
What's difficult about this typing? Is it dependent upon configured types or something?

Reply:
It is yeah.